packages/react-components/review/react-components.api.md

```diff
@@ -359,6 +359,18 @@ export interface GridLayoutProps {
     styles?: BaseCustomStyles;
 }
 
+// @public
+export interface GridLayoutStyles extends BaseCustomStyles {
+    children?: IStyle;
+}
+
+// @public
+export interface HorizontalGalleryStyles extends BaseCustomStyles {
+    children?: IStyle;
```

Comment:
Does "children" mean videoTile? Or something else? In html, term "children" means every dom element under a parent - if that is videoTile we can call it videoTile or something more specific

Reply:
And "children" makes me expect an array of IStyles for types as well, which doesn't match what we show here

Reply:
Good question. GridLayout and HorizontalGallery each take in children prop of every dom element so it is not only limited to VideoTile. In this case, I am using applying one IStyle to all **direct** children. It would be hard to apply an array of styles to an array of children. what if the arrays don't have the same size. I think this is the most straightforward way but intered in what others think as well.

Reply:
Maybe a quick storybook example could explain how this works, but yep, if that is all children elements, it makes sense to name it as children